packages/backend/src/assets/inference-images.json

```diff
@@ -3,8 +3,8 @@
     "default": "ghcr.io/containers/whispercpp@sha256:6c529656529da7aba851b6ab2d0653f23b77b9ca3c11cb46db47ff6dbd6d5e7c"
   },
   "llamacpp": {
-    "default": "ghcr.io/containers/podman-desktop-extension-ai-lab-playground-images/ai-lab-playground-chat:f531f3f4a899b16071310511d19b3913fd0515b8",
-    "cuda": "ghcr.io/containers/podman-desktop-extension-ai-lab-playground-images/ai-lab-playground-chat-cuda:f531f3f4a899b16071310511d19b3913fd0515b8",
-    "vulkan": "ghcr.io/containers/podman-desktop-extension-ai-lab-playground-images/ai-lab-playground-chat-vulkan:f531f3f4a899b16071310511d19b3913fd0515b8"
+    "default": "ghcr.io/containers/podman-desktop-extension-ai-lab-playground-images/ai-lab-playground-chat:0.5.1",
```

Comment:
we are losing some security by using a tag instead of the sha256 here

Reply:
Whick kind of security as I used tag because it is more user and support friendly but eager to change

Reply:
There is nothing preventing us from overwriting a tag in a registry, or a malicious user redirecting to another registry.

Adding the sha256 ensure the image pulled is the image we expect. We can use the following syntax 

`ghcr.io/containers/podman-desktop-extension-ai-lab-playground-images/ai-lab-playground-chat:0.5.1@sha256:20734e9d60f047d27e4c9cf6a3b663e0627d48bd06d0a73b968f9d81c82de2f1` where you have both the tag, and the sha256, so it is user friendly (to know which version it is)

![Screenshot from 2024-10-14 16-31-30](https://github.com/user-attachments/assets/8ffd3296-b54c-49fa-8434-eed684e80bd8)

The sha256 is available on the GitHub  page, so it is not hard to change in my opinion, but this is the not the direction we want to take I will not be blocking this PR

Reply:
I like the version@sha256 notation because it gives us both pros but it is rejected by Podman CLI so testing it from Podman Desktop